targets/TARGET_Cypress/TARGET_PSOC6/psoc6csp/hal/include/cyhal_hw_resources.h

```diff
@@ -65,7 +65,6 @@ typedef enum
     CYHAL_RSC_OPAMP,     /*!< Opamp */
     CYHAL_RSC_PDM,       /*!< PCM/PDM communications block */
     CYHAL_RSC_SMIF,      /*!< Quad-SPI communications block */
-    CYHAL_RSC_TRNG,      /*!< Hardware random number generator */
```

Comment:
Was it intentional to remove CRC and TRNG? Does the hardware have a TRNG or CRC?

Reply:
CRC and TRNG are subresources of CRYPTO hardware so now it were moved into resource subtypes

Reply:
CRC and TRNG are parts of the same hardware resource CRYPTO so it were moved to resources subtype

Reply:
OK